packages/components/inputs/rich-text-input/src/rich-text-input.stories.tsx

```diff
@@ -0,0 +1,113 @@
+
+const meta: Meta<typeof RichTextInput> = {
+  title: 'Form/Inputs/RichTextInput',
+  // @ts-ignore
+  component: RichTextInput,
+};
+export default meta;
+
+type Story = StoryFn<typeof RichTextInput>;
+
+const initialValue =
+  '<h1>Heading</h1><p>This is a paragraph.<br/><br/>New sentence, <strong>same</strong> paragraph, but with <u>soft</u>-linebreak (<em>Shift + Enter</em>).</p>';
+
+// @ts-ignore
+export const BasicExample: Story = (args: TRichTextInputProps) => {
+  const [value, setValue] = useState(initialValue);
+  return (
+    <RichTextInput
+      {...args}
+      value={value}
+      onChange={(e) => setValue(e.target.value)}
+    />
+  );
+};
+
+BasicExample.args = {
+  defaultExpandMultilineText: true,
+};
+
+/** This demo allows to modify the intial input, do changes and verify if the html-output matches the expectations. */
+export const PlaygroundExample: Story = () => {
+  const onClickExpand = useCallback(() => {
+    // eslint-disable-next-line no-alert
+    alert('Expand');
+  }, []);
+
+  const onBlur = useCallback(() => console.log('onBlur'), []);
+  const onFocus = useCallback(() => console.log('onFocus'), []);
+  const ref = useRef(null);
+  const [value, setValue] = useState(initialValue);
+  const [resetValue, setResetValue] = useState(initialValue);
+  const onChange = useCallback(
+    (event) => {
+      setValue(event.target.value);
+    },
+    [setValue]
+  );
+  const onResetValueChange = useCallback(
+    (event) => {
+      setResetValue(event.target.value);
+    },
+    [setResetValue]
+  );
+  const handleReset = () => {
+    ref.current?.resetValue(resetValue);
+  };
+
+  return (
+    <Spacings.Stack scale="l">
+      <CollapsiblePanel
+        header="Set initial value"
+        horizontalConstraint="scale"
+        isDefaultClosed
+      >
+        <Constraints.Horizontal max="scale">
+          <Spacings.Stack scale="m">
+            <textarea
+              defaultValue={resetValue}
+              onChange={onResetValueChange}
+              rows={4}
+            />
+            <Constraints.Horizontal max="auto">
+              <PrimaryButton
+                label="Reset"
+                onClick={handleReset}
+                size="medium"
+              />
+            </Constraints.Horizontal>
+          </Spacings.Stack>
+        </Constraints.Horizontal>
+      </CollapsiblePanel>
+      <RichTextInput
+        name={'test-name'}
+        onBlur={onBlur}
+        onFocus={onFocus}
+        defaultExpandMultilineText={true}
+        placeholder={'Placeholder'}
+        showExpandIcon={false}
+        onClickExpand={() => {
+          onClickExpand();
+          return true;
+        }}
+        hasError={false}
+        hasWarning={false}
+        isDisabled={false}
+        isReadOnly={false}
+        ref={ref}
+        onChange={onChange}
+        value={value}
+      />
+      <Text.Headline as="h3">Output</Text.Headline>
+      <pre>{value}</pre>
```

Comment:
```suggestion
      <pre style={{ textWrap: 'auto' }}>{value}</pre>
```

https://github.com/user-attachments/assets/aba230a6-077d-46cf-afab-325ae062558b